config.yml

```diff
@@ -328,6 +318,21 @@ tokens:
     comment: "a separator between words in a list"
   - name: __END__
     comment: "marker for the point in the file at which the parser should stop"
+  - name: KEYWORD_IF_MODIFIER
+    value: 32768
```

Comment:
Can we add a comment about the YP_MODIFIER_FLAG (or something to do with 2^15) here too so that they don't seem random?